olmo/config.py

```diff
@@ -1217,6 +1217,11 @@ class TrainConfig(BaseConfig):
     Path to cache directory of HF datasets saved with `datasets.save_to_disk`.
     """
 
+    module_outputs_save_steps: Optional[List[int]] = None
+    """
+    Outputs of model submodules are saved during the provided steps.
```

Comment:
Can you add a note here about how to use the script to view the results?

Reply:
https://github.com/allenai/OLMo/pull/690/commits/81c07c6c75210c25d7255b7e5665a7076d04027e I've mentioned the script for now. I intend to add a doc describing debugging with this soon.